src/bundle.c

```diff
@@ -339,23 +339,23 @@ int add_subscriptions(struct list *bundles, int current_version, struct manifest
 			continue;
 		}
 
-retry_manifest_download:
+	retry_manifest_download:
```

Comment:
I would prefer labels always start on column 1. I hope that the rest of the code agrees. :)